<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEditVariable</class>
 <widget class="QDialog" name="FormEditVariable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>238</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add Variable</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Calc. Value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Input Value</string>
     </property>
     <property name="buddy">
      <cstring>m_cmbType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="buddy">
      <cstring>m_txtName</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="buddy">
      <cstring>m_txtDescription</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="m_cmbType">
     <property name="toolTip">
      <string>Select type of variable.</string>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <item>
      <property name="text">
       <string>INTEGER</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FLOAT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BOOLEAN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>STRING</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COMPLEX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VOID</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="LineEdit" name="m_txtName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string>Name of variable</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="LineEdit" name="m_txtCalculatedValue">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Calculated value</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="LineEdit" name="m_txtDescription">
     <property name="placeholderText">
      <string>Description of variable</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="m_btnBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="LineEdit" name="m_txtValue">
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string>Input value here</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="m_lblInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_txtName</tabstop>
  <tabstop>m_txtValue</tabstop>
  <tabstop>m_txtDescription</tabstop>
  <tabstop>m_btnBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_btnBox</sender>
   <signal>rejected()</signal>
   <receiver>FormEditVariable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
